src/contexts/AuthContext.tsx

```
@@ -178,7 +169,7 @@
     };
 
     initializeAuth();
-  }, []);
+  }, [refreshAuth]);
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)